src/main/java/com/twitter/kinesis/utils/Environment.java

```diff
@@ -36,70 +36,69 @@ public void logProperties() {
   }
```

Comment:
What's the state of that repo? Should we even have this one if we've copied the code into offerpop/gnip-kinesis-connector ?